mockintosh/servers.py

```diff
@@ -39,15 +42,32 @@ def load(self):
 
         for port, services in port_mapping.items():
             rules = []
+            ssl = False
+            for service in services:
+                if ssl:
+                    break
+                ssl = service.get('ssl', False)
+
+            protocol = 'https' if ssl else 'http'
+            ssl_options = {
+                "certfile": path.join(__location__, 'ssl/cert.pem'),
```

Comment:
If you use join, use it for all components, like `path.join(__location__, 'ssl', 'cert.pem'),`

Reply:
@undera fixed with https://github.com/up9inc/mockintosh/commit/34b0c0c21a4b0e97ad48b4663317ff4d28cce224